### PR TITLE
MGMT-3371 Documentation on adding support for a new OLM operator

### PR DESCRIPTION
**Pull request by jakub-dzon**

### CODE REVIEW

**Comment by jakub-dzon** [comment]
@gamli75, @tsorya - please have a look.

File: docs/dev/olm-operator-plugins.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      		If(AreOcsRequirementsSatisfied), If(AreLsoRequirementsSatisfied))
+      ```
+ 1. Implement the [`Operator` interface](../../internal/operators/api/api.go)
+ 1. Plug the new `Operator` implementation in the [OperatorManager constructor](../../internal/operators/builder.go):
+    ```go
+    func NewManager(log logrus.FieldLogger) Manager {
+    	return NewManagerWithOperators(log, lso.NewLSOperator(), ocs.NewOcsOperator(log))
+    }
+    ```
+ 1. Make sure all the tests are green
```
Comment (gamli75) [inline docs/dev/olm-operator-plugins.md:90]: Add specific tests, such as validation tests that related to the new OLM

**Review by gamli75 — COMMENTED** [review]
Looks good to me

File: docs/dev/olm-operator-plugins.md (hunk 1 above)
Comment (jakub-dzon) [reply docs/dev/olm-operator-plugins.md:91]: Added.

**Review by gamli75 — APPROVED** [review]

**Pull request merged**